src/drivers/mct/main/seq_io_mod.F90

```diff
@@ -307,30 +307,17 @@ end subroutine seq_io_enddef
 
 !===============================================================================
 
-character(len=10) function seq_io_date2yyyymmdd (date)
+character(len=24) function seq_io_date2yyyymmdd (date)
```

Comment:
Why have a seperate function instead of locating in shr_cal_mod?

Reply:
Because I had already spent a few hours on this and just wanted to be done. Feel free to move into shr_cal_mod if you'd like.